partcad-cli/src/partcad_cli/click/command_test.py

```
@@ -0,0 +1,18 @@
+from click.testing import CliRunner
+from partcad_cli.click.command import cli
+from partcad_cli import __version__
+import logging
+from typing import Iterator
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)